lib/packages/shared-utils/date-helper.ts

```diff
@@ -1,26 +1,27 @@
-import * as dateFns from "date-fns";
+import { format, add } from "date-fns";
+
 export function formatDate(date: number | null | undefined) {
   if (!date || date === undefined) {
     return "Pending";
-  } else {
-    return dateFns.format(date, "MMMM d, yyyy");
   }
+
+  return format(date, "MMMM d, yyyy");
 }
 
+export const isDST = (date: Date): boolean => {
+  const januaryOffset = new Date(date.getFullYear(), 0, 1).getTimezoneOffset();
+  const julyOffset = new Date(date.getFullYear(), 6, 1).getTimezoneOffset();
+  return date.getTimezoneOffset() !== Math.max(januaryOffset, julyOffset);
+};
+
 export function formatNinetyDaysDate(date: number | null | undefined): string {
-  if (!date || date === undefined) {
+  if (!date) {
     return "Pending";
   }
 
-  const isDST = (date: number): boolean => {
-    const jan = new Date(date).getTimezoneOffset();
-    const jul = new Date(new Date(date).setMonth(6)).getTimezoneOffset();
-    return new Date(date).getTimezoneOffset() < Math.max(jan, jul);
```

Comment:
This might be our culprit.  Since the logic is changing in how tme is calculated line 21's conditional is changing.  It goes from using the timezone offset, which probalby cares more about daylight savings, to adding just days which does.  The test are always saying "yes we care about daylight savings now" and possibly the logic before didnt so it would alwasy do est.